### PR TITLE
imagine.lua

### PR DESCRIPTION
**Pull request by vitawrap**

### CODE REVIEW

File: caillef/imagine.lua [hunk 1]
```diff
@@ -269,22 +282,28 @@ end
...
 
 Pointer.Down = function(pe)
-	menu:hide()
 end
 
 Pointer.Up = function(pe)
 end
 
-Pointer.Drag = function(pe)
+Pointer.Drag2Begin = function(pe)
```
Comment (aduermael) [inline caillef/imagine.lua:290]: Please remove empty functions! 🙂

File: caillef/imagine.lua [hunk 2]
```diff
@@ -269,22 +282,28 @@ end
 -- UI CODE
 
 Pointer.Down = function(pe)
```
Comment (aduermael) [inline caillef/imagine.lua:284]: Please remove empty functions! 🙂

File: caillef/imagine.lua [hunk 3]
```diff
@@ -269,22 +282,28 @@ end
...
-	menu:hide()
 end
 
 Pointer.Up = function(pe)
```
Comment (aduermael) [inline caillef/imagine.lua:287]: Please remove empty functions! 🙂

**Comment by vitawrap** [comment]
Adressed the review and hopefully removed all of the dangling deprecated stuff!

**Review by aduermael — CHANGES_REQUESTED** [review]
The quality of the game has clearly improved! Well done @vitawrap! 
Still a few issues, and not all of them really due to that PR: 

**Clicking on a generated image doesn't show info:**

Screenshot 2023-07-21 at 07 42 30

**Create Image buttons are not positioned like text inputs (they should be):**

Screenshot 2023-07-21 at 07 40 19


This is for @ArthurVoxowl (and I), `TextBubble` may need to be fixed following the PR about texts.

Screenshot 2023-07-21 at 07 39 03

Also, using semi transparent color for texts created depth / blending issues in the UI:

Screenshot 2023-07-21 at 07 40 31

**Review by aduermael — APPROVED** [review]
LGTM! 
Merging now, addressing my comments in following PR. 🙂

**Pull request merged**